cmd/install

```diff
@@ -16,7 +16,13 @@
 
 echo Installing application dependencies
 
-TESTRUN_DIR=/usr/local/testrun
+if [[ -z "${TESTRUN_DIR}" ]]; then
+  TESTRUN_DIR=/usr/local/testrun
+else
+  TESTRUN_DIR="${TESTRUN_DIR}"
```

Comment:
else statement unnecessary as it is just setting the same variable with the existing value

Reply:
The docs are shortly going to be updated to state installing to a custom directory will require the following command: sudo TESTRUN_DIR=custom/dir cmd/install. This will allow using the package installer and custom directory

Reply:
Could we make this a command line arg instead?  Something like --install-dir.  This might be a cleaner option and then update this script to accept the args as necessary.

Reply:
I have just added the option for -l to instal locally